src/error.rs

```diff
@@ -37,12 +49,19 @@ impl Error {
         }
     }
 
+    /// Does nothing if self.pointer is NULL; if self.pointer is non-NULL, then *self.pointer must be NULL.
+    /// A new GError is created and assigned to *self.pointer .
```

Comment:
Discussions of pointers are likely to confuse. Unfortunately, simply copypasting the documentation doesn't always make sense.